config/environments/production.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.configure do
   # Specify environment specific hostname and protocol
-  config.hostname = ENV["HOSTNAME"]
+  config.hostname = "#{ENV["HOSTNAME"]}/api"
```

Comment:
actually I don't think I should have to hard code this at all -- this should be in the devops configuration and HOSTNAME is the HOSTNAME + any additional base path that they add to the HOSTNAME. @ayaleloehr what do you say I remove this and we just add it to the ansible scripts? And they put in a note to update when we switch to api.vets.gov seeing as it is entirely outside of our control in the first place (why its in ENV to begin with).